UPGRADE.md

```diff
@@ -10,8 +10,10 @@ does not have any particular instructions.
 
 ### Helm
 
-Under `cni.image`, the default values for `repository` and `registry` have been
+- Under `cni.image`, the default values for `repository` and `registry` have been
 changed to agree with the other `image` values.
+- We are deprecating `kuma.io/builtindns` and `kuma.io/builtindnsport` annotations in favour or the clearer `kuma.io/builtin-dns` and `kuma.io/builtin-dns-port` the code is backward compatible
```

Comment:
> the code is backward compatible

What does this mean? That no action is required but users _should_ migrate?

Reply:
Yes I guess this wording is wrong. Do you have an idea on how to rephrase?